recipes/nifti_clib/all/conanfile.py

```diff
@@ -0,0 +1,160 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class NiftiClibConan(ConanFile):
+    name = "nifti_clib"
+    description = "C libraries for NIFTI support"
+    license = "LicenseRef-LICENSE"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/NIFTI-Imaging/nifti_clib"
+    topics = ("image")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_nifti2": [True, False],
+        "use_cifti": [True, False],
+        "use_fslio": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "use_nifti2": True,
+        "use_cifti": False,  # seems to be beta?
+        "use_fslio": False  # Note in CMakeLists.txt: "If OFF, The copyright of this code is questionable for inclusion with nifti."
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
```

Comment:
```suggestion
    def export_sources(self):
        export_conandata_patches(self)

```

Can be removed as there are no patches :)